snap/snapcraft.yaml

```diff
@@ -0,0 +1,67 @@
+name: graphs # you probably want to 'snapcraft register <name>'
+base: core22 # the base snap is the execution environment for this snap
+adopt-info: graphs
+grade: stable # must be 'stable' to release into candidate/stable channels
+confinement: strict # use 'strict' once you have the right plugs and slots
+compression: lzo
+license: GPL-3.0
+architectures:
+  - build-on: amd64
+  - build-on: arm64
+
+slots:
+  graphs:
+    interface: dbus
+    bus: session
+    name: se.sjoerd.Graphs
+
+parts:
+  graphs:
+    # See 'snapcraft plugins'
+    plugin: meson
+    source: https://github.com/Sjoerd1993/Graphs.git
+    #source-tag: 'v1.5.2'
+    meson-parameters:
+      - --prefix=/snap/graphs/current/usr
+      - --buildtype=release
+    build-packages:
+      - blueprint-compiler
+    override-pull: |
+      craftctl default
+      patch -p1 < $CRAFT_PROJECT_DIR/snap/graphs.patch
+    override-build: |
+      craftctl default
+      pip install --prefix=$CRAFT_PART_INSTALL/usr matplotlib==3.7.1
+      pip install --target=$CRAFT_PART_INSTALL/usr/lib/python3/dist-packages scipy==1.10.1
```

Comment:
just checked this, in a "regular" pip install packages get installed to site-packages (this is what we use on the flatpak side as well as the graphs module itself). Any reason these aren't installed there?